### ISSUE TITLE
BoF Theme

### ISSUE THREAD

**Issue by afrind** [afrind/moqbof#5]
"What practical protocol glue to make stuff (media) work over QUIC is missing?" was a suggested theme.

**Comment by SpencerDawkins** [comment]
I agree, but I suspect that the some "media stuff" is pretty close to working over QUIC, while other stuff is quite a bit further from working. This seems like a post-BOF question to me.

**Comment by afrind** [comment]
> "What practical protocol glue to make stuff (media) work over QUIC is missing?" 

Media over QUIC captures too many possible use cases to be used in a question like this.  "What protocol glue to make <specific use case(s)> work is missing?" is probably more apt.

We aren't interested in moving media over QUIC just because it's buzzy.   QUIC as a transport abstraction provides a bunch of features out of the box that make it very simple to do some very powerful things.
